CHANGELOG.md

```diff
@@ -4,28 +4,92 @@
+`teleport-update` will become the new standard way of installing Teleport as it
+always picks the appropriate Teleport edition (Community vs Enterprise), the
+cluster's desired version, and the correct Teleport variant (e.g. FIPS-compliant
 cryptography).
 
-You can find more information about the feature in [our documentation]().
-
 ### Package layout changes
 
-Starting with 17.3.0, the Teleport DEB and RPM packages, notably used by the `apt`, `yum`, `dnf` and `zypper` package
-managers, will place the Teleport binaries in `/opt/teleport` instead of `/usr/local/bin`.
+Starting with 17.3.0, the Teleport DEB and RPM packages, notably used by the
+`apt`, `yum`, `dnf` and `zypper` package managers, will place the Teleport
+binaries in `/opt/teleport` instead of `/usr/local/bin`.
+
+The binaries will be symlinked to their previous location, no change should be
+required in your scripts or systemd units.
+
+This change allows us to do automatic updates without conflicting with the
+package manager.
+
+### Delegated joining for Oracle Cloud Infrastructure
+
+Teleport agents running on Oracle Cloud Infrastructure (OCI) are now able to
+join the Teleport cluster without a static join token.
+
+### Stable UIDs for host-user creation
+
+Teleport now provides the ability to create host users with stable UIDs across
+the entire Teleport cluster.
+
+### VNet for Windows
+
+Teleport's VNet feature are now available for Windows, allowing users to access
+TCP applications protected by Teleport as if they were on the same network.
+
+### Improved GitHub Proxy enrollment flow
 
-The binaries will be symlinked to their previous location, no change should be required in your scripts or systemd units.
+Teleport web UI now provides wizard-like guided enrollment flow for the new
+GitHub Proxy integration.
+
+### AWS Identity Center integration improvements
+
+AWS Identity Center integration now supports using IAM authentication instead of
+OIDC (useful for private clusters) and a hybrid setup that allows to use another
+IdP as external identity source.
+
+### Okta integration improvements
+
+Teleport Okta integration now provides updated guided enrollment flow and will
+allow updating integration settings (such as sync configuration or group
+filters) without having to recreate the integration.
+
+Note that the new enrollment flow uses OAuth authentication method instead of
+API tokens. If the Okta integration is installed on v17.3 and the cluster is
+downgraded the Okta plugin must be reinstalled to ensure proper functionality.
+
+### Readiness endpoint changes
+
+The Auth Service readiness now reflects the connectivity from the agent to the
+backend storage, and the Proxy Service readiness also reflects its connectivity
+to the Auth Service API. In case of Auth or backend storage failure, the agents
+will now turn unready. This change ensures that control plane components can be
```

Comment:
```suggestion
to the Auth Service API. In case of Auth or backend storage failure, the auth and proxy pods
will now turn unready. This change ensures that control plane components can be
```

Reply:
I meant agents as in the teleport processes running auth or proxy but you're right, i'll change the wording.